rootfs/deis/gconf.py

```diff
@@ -1,4 +1,6 @@
 import os
+import faulthandler
+faulthandler.enable()
```

Comment:
Yeah, haven't looked at if I can replace the `traceback` below but we will see in the future